v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/likes.kt

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 Shimizu Yasuhiro (yshrsmz)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.codingfeline.twitter4kt.v1.api.statuses
+
+import com.codingfeline.twitter4kt.core.ApiResult
+import com.codingfeline.twitter4kt.core.apiUrl
+import com.codingfeline.twitter4kt.core.util.Twitter4ktInternalAPI
+import com.codingfeline.twitter4kt.core.util.appendNotNulls
+import com.codingfeline.twitter4kt.v1.api.getInternalListResponse
+import com.codingfeline.twitter4kt.v1.model.status.Tweet
+import io.ktor.http.Parameters
+
+/**
+ * Get's a non-protected user's timeline of their likes (previously favorites). This can be requested either using their Twitter ID
+ * or by using their screenname. One must be passed in for the call to succeed. This will return the full 280 characters
```

Comment:
Not passing userId/screenName results in authenticated user's likes, so we should cover this case.

Reply:
That's interesting I didn't realize that. Yep!